src/components/Button/styled/button.js

```diff
@@ -0,0 +1,246 @@
+import styled from 'styled-components';
+import {
+    COLOR_WHITE,
+    COLOR_BRAND_ACTIVE,
+    COLOR_BRAND,
+    COLOR_GRAY_1,
+    COLOR_GRAY_2,
+    COLOR_GRAY_3,
+    COLOR_GRAY_4,
+    COLOR_ERROR,
+    COLOR_ERROR_ACTIVE,
+    COLOR_SUCCESS,
+    COLOR_SUCCESS_ACTIVE,
+} from '../../../styles/colors';
+import { FONT_SIZE_HEADING_SMALL } from '../../../styles/fontSizes';
+import { SHADOW_OUTLINE, SHADOW_5, SHADOW_1 } from '../../../styles/shadows';
+
+const StyledButton = styled.button`
+    align-items: center;
+    display: inline-flex;
+    font-size: ${FONT_SIZE_HEADING_SMALL};
+    justify-content: center;
+    position: relative;
+    background: transparent;
+    background-clip: border-box;
+    border: 1px solid transparent;
+    border-radius: 100px;
+    line-height: 2.375rem;
+    text-decoration: none;
+    color: ${COLOR_BRAND};
+    padding: 0 1rem;
+    cursor: pointer;
+    white-space: normal;
+    user-select: none;
+    text-align: center;
+    vertical-align: middle;
+    transition: border 0.15s linear;
+
+    &:hover,
+    &:focus,
+    &:active,
+    &:visited {
+        text-decoration: none;
+    }
+
+    &:hover,
+    &:focus {
+        color: ${COLOR_BRAND_ACTIVE};
+    }
+
+    &:focus {
+        outline: 0;
+        box-shadow: ${SHADOW_OUTLINE};
+    }
+
+    &:active {
+        color: ${COLOR_BRAND_ACTIVE};
+        transform: scale(0.95);
+        transition: all 0.2s ease;
+    }
+
+    &[disabled] {
+        color: ${COLOR_GRAY_2};
+    }
+
+    &[disabled] * {
+        cursor: default;
+        pointer-events: none;
+    }
+
+    ${props =>
+        props.variant === 'neutral' &&
+        `
+            background-color: ${COLOR_WHITE};
+            border: 1px solid ${COLOR_GRAY_2};
+            color: ${COLOR_BRAND};
+
+            &:hover,
+            &:focus,
+            &:active {
+                background-color: ${COLOR_GRAY_1};
+            }
+
+            &[disabled] {
+                background-color: ${props.isLoading ? 'COLOR_WHITE' : 'transparent'};
```

Comment:
use COLOR_WHITE constant without ' ' because here it converts to this css:
```background-color: COLOR_WHITE ``` and there is not COLOR_WHITE color so use:
```background-color: ${props.isLoading ? COLOR_WHITE : 'transparent'};```